src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -2003,33 +2003,37 @@ class MavlinkStreamLocalPositionNEDCOV : public MavlinkStream
 
 	bool send(const hrt_abstime t)
 	{
-		struct estimator_status_s est = {};
+		estimator_status_s est;
 
 		if (_est_sub->update(&_est_time, &est)) {
-			mavlink_local_position_ned_cov_t msg = {};
 
-			msg.time_usec = est.timestamp;
-			msg.x = est.states[0];
-			msg.y = est.states[1];
-			msg.z = est.states[2];
-			msg.vx = est.states[3];
-			msg.vy = est.states[4];
-			msg.vz = est.states[5];
-			msg.ax = est.states[6];
-			msg.ay = est.states[7];
-			msg.az = est.states[8];
-
-			for (int i = 0; i < 9; i++) {
-				msg.covariance[i] = est.covariances[i];
-			}
+			// HACK for LPE, this is completely wrong for EKF2
```

Comment:
This is not needed anymore after https://github.com/PX4/Firmware/pull/9895

Reply:
https://github.com/PX4/Firmware/pull/9895/files#diff-478580be4572c29aaaa7e5861078259eR648